### PR TITLE
Move logic for canceling books reservation outside of the Web application

### PR DESCRIPTION
**Pull request by rafalskowronski**

### CODE REVIEW

File: cron/src/cron_job [hunk 1]
```diff
@@ -0,0 +1 @@
+* * * * * root . $HOME/.env.sh; /usr/local/bin/python /app/update_reservation_status.py > /proc/1/fd/1 2>/proc/1/fd/2
```
Comment (Jahenstein) [inline cron/src/cron_job]: Running this script every minute might corrupt database. It is plausible that script and user will attempt to change book status at the same time. Previously cron executed script once a day, to minimize risk of such event.

```suggestion
0 3 * * * root . $HOME/.env.sh; /usr/local/bin/python /app/update_reservation_status.py > /proc/1/fd/1 2>/proc/1/fd/2
```

File: cron/src/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+pyflakes==2.1.1
+pyparsing==2.4.0
+pytest==4.6.3
+python-dateutil==2.8.0
+python-dotenv==0.10.3
+six==1.12.0
+SQLAlchemy==1.3.5
+SQLAlchemy-Utils==0.34.0
+wcwidth==0.1.7
+zipp==0.5.1
```
Comment (Jahenstein) [inline cron/src/requirements.txt]: Majority of those are not needed, if I understand correctly only these three are needed:

freezegun==0.3.12
pytest==4.6.3
SQLAlchemy==1.3.5

Comment (rafalskowronski) [reply cron/src/requirements.txt]: Requirements files are typically used to hold the result of `pip freeze`. `pip freeze` on the other hand outputs all installed packages, so as a result requirements file contains all packages, not only the top ones.
More [here](https://pip.pypa.io/en/stable/user_guide/#requirements-files).

File: cron/src/cron_job (hunk 1 above)
Comment (rafalskowronski) [reply cron/src/cron_job]: Parallel access to DB should be studied during architecture design. Running script rarely should not be the solution for existing design gap. I've introduced 'SERIALIZABLE' DB isolation level in cron job and also in web app (see PR #277) as a solution for problems concerning DB concurrent access.

File: cron/src/update_reservation_status.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            copy
+            .update()
+            .where(copy.c.id == bindparam('copy_id'))
+            .values(available_status=BookStatus.RETURNED),
+            bind_items
+        )
+
+    print("[{}] Cancelled reservation for library item id: {}"
+          .format(datetime.now(),
+                  ', '.format([item[1] for item in items])))
```
Comment (Jahenstein) [inline cron/src/update_reservation_status.py]:
```suggestion
    print("[{}] Cancelled reservation for library item id: {}"
          .format(datetime.now(), [item[1] for item in items]))
```
Beside typo in print everyth

**Pull request merged**